### ISSUE TITLE
Restore to backups

### ISSUE THREAD

**Issue by IAmLythixIAmDumb** [pterodactyl/panel#2196]
**Is your feature request related to a problem? Please describe.**
I think it's annoying that with backups, you can only download them but not restore to them directly in the panel.

**Describe the solution you'd like**
I think you should make it so that you can restore backups by having wings extracting the files from the backup, and moving the extracted files into the server directory essentially restoring to the backup. **Should probably delete the existing files before putting the backed up files in**

**Describe alternatives you've considered**
Allow users to upload a backup they've downloaded in the past to restore from that. (May not be possible on servers running some eggs like minecraft as they take a lot of space and will reach the web upload limit on most servers.)

**Additional context**
Basically allowing to restore from backups users upload (possibly) and existing backups (that exist on wings already.)

**Comment by Sir3lit** [comment]
Possibly duplicate of #2016?

**Comment by IAmLythixIAmDumb** [comment]
> Possibly duplicate of #2016?

Possibly, but this includes info about how it would be done. Explained in the solution.

**Comment by parkervcp** [comment]
I don't care if it describes that it includes a how it would be done. It's a duplicate in my eyes. You can explain how you think it "should" be done in the other one.

**Comment by DaneEveritt** [comment]
@IAmLythixIAmDumb would you mind just adding your notes to #2016 so we can keep it all contained in one issue rather than multiple almost-the-same type issues? 👍

**Comment by IAmLythixIAmDumb** [comment]
> @IAmLythixIAmDumb would you mind just adding your notes to #2016 so we can keep it all contained in one issue rather than multiple almost-the-same type issues?

Sure, infact, I didn't even find the first issue when I made this one. Sorry for clogging your issue tracker XD
